lib/src/components/NxTransferList/NxTransferList.tsx

```diff
@@ -9,17 +9,17 @@ import classnames from 'classnames';
 import { faPlusCircle, faTimesCircle } from '@fortawesome/free-solid-svg-icons';
 import { toLower, filter, includes, groupBy, partial, identity, prop, pipe } from 'ramda';
 
-import { Props, TransferListItemProps, DataItem, FilterFn } from './types';
+import { Props, propTypes, TransferListItemProps, DataItem, FilterFn } from './types';
 import NxFilterInput from '../NxFilterInput/NxFilterInput';
 import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
 import NxFieldset from '../NxFieldset/NxFieldset';
 import NxOverflowTooltip from '../NxTooltip/NxOverflowTooltip';
 
-export { Props, DataItem } from './types';
+export { Props, propTypes, DataItem } from './types';
```

Comment:
No need to export this since it's attached to the component function

Reply:
Ah, I was using NxTooltip as a reference and it exported them.
Removed in this commit:
8df61c7ba4c2ad10415a869de9993da4bfe5b9af